rpmconf/Spec/Scripts/zimbra-proxy.post

```diff
@@ -18,8 +18,21 @@
 
 chown -R zimbra:zimbra /opt/zimbra/conf/nginx
 
-chown root:zimbra /opt/zimbra/common/sbin/nginx
-chmod 750 /opt/zimbra/common/sbin/nginx
+if [ -x /opt/zimbra/log/nginx.log ]; then
+    chown zimbra:zimbra /opt/zimbra/log/nginx.log
+    chmod 644 /opt/zimbra/log/nginx.log
+fi
+
+if [ -x /opt/zimbra/log/nginx.access.log ]; then
+    chown zimbra:zimbra /opt/zimbra/log/nginx.access.log
+    chmod 644 /opt/zimbra/log/nginx.access.log
+fi
```

Comment:
As in the other PR, should these tests be using `-x`  to check the executable flag, or `-f` or `-e` just to test for existence?

Reply:
Fixed